ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,12 +10,24 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
+    private static final By NOTE = By.id("w10");
+    String note = "this is edited note";
+    private static final By REMOVE_DIAGNOSIS = By.cssSelector("#display-encounter-diagnoses-container > ul:nth-child(7) > li > span > i");
```

Comment:
Do you want this to drop both primary and secondary at the same time, this current would only pick the secondary one? 
Rather use something more reliable such as;
`i[data-ng-click='removeDiagnosis(d)']`
or
`.icon-remove.delete-item`

Reply:
Resolved thanks